classes/jplain.lua

```diff
@@ -17,13 +17,21 @@ function jplain:_j_common ()
   self:loadPackage("font-fallback")
   self:loadPackage("hanmenkyoshi")
   self:registerPostinit(function (class)
-      class:bidiDisableTypesetter(SILE.typesetter)
-      class:bidiDisableTypesetter(SILE.defaultTypesetter)
-      SILE.call("font:add-fallback", { family = "Noto Sans CJK JP" })
-    end)
+    class:bidiDisableTypesetter(SILE.typesetter)
+    class:bidiDisableTypesetter(SILE.defaultTypesetter)
+  end)
   self.defaultFrameset.content.tate = self.options.layout == "tate"
   self:declareHanmenFrame("content", self.defaultFrameset.content)
   SILE.settings:set("document.parindent", SILE.nodefactory.glue("10pt"))
+  if SILE.settings:get("document.language") ~= "ja" then
```

Comment:
Nice work with the deprecation warning for documents using the old style. I wonder though if we might be able to clarify the scopes here even further. I would be among the many to assume `jplain` probably defaulted to Japaneese. What if we renamed this class to `cjkplain` or something like that (you might know better what an appropriate umbrella name would be) and then make a new `jplain` class that inherits from it and just adds the default to Japanese language & fonts?

Reply:
That's a good idea. I suggest `tplain`. This has precedent from pLaTeX, the t is for `tate`.

That way in the far future this can be used for Mongolian too!!

Reply:
PS I'm aware that this layout isn't always used to make TTB docs of course. I am just suggesting the name to mean "`tate`-_capable_ plain document".

I'd like to see this file go away as much as anyone, I think plain old `plain` should support `tate`, but that'll require getting on board with my thoughts re what to do about BiDi and `latin-in-tate`  :) so this class'll be around a while.

Reply:
I'm not knowledgeable enough to weigh in on BiDi in TTB, so that decision might get put off a bit. Honestly I do think loading the tate package being viable in the default class makes sense too, but in the mean time lets shuffle the cheese and go with `tplain` that makes no language assumptions and `jplain` that does. That will also make this whole PR a *non* breaking change, right?

Reply:
As far as I can see most of this is resolved and for the bit that isn't we have other issues tracking them.